sdk/identity/identity/README.md

```diff
@@ -1,6 +1,6 @@
 ## Azure Identity client library for JavaScript
 
-The Azure Identity library provides Azure Active Directory token authentication support across the Azure SDK. It provides a set of [TokenCredential](https://docs.microsoft.com/javascript/api/@azure/core-auth/tokencredential) implementations which can be used to construct Azure SDK clients which support AAD token authentication.
+The Azure Identity library provides [Azure Active Directory (AAD)](https://azure.microsoft.com/services/active-directory/) token authentication through a set of convenient [TokenCredentials](https://docs.microsoft.com/javascript/api/@azure/core-auth/tokencredential). It enables Azure SDK clients to authenticate with Azure AAD, while also allowing other JavaScript and TypeScript applications to authenticate with AAD work and school accounts, Microsoft personal accounts (MSA), and other Identity providers through the [Azure AD B2C](https://azure.microsoft.com/services/active-directory/external-identities/b2c/) service.
```

Comment:
Let's direct the customer to docs instead. The marketing pages you've referenced are better for customers who haven't yet looked at this README file.

```suggestion
The Azure Identity library provides [Azure Active Directory (AAD)](https://docs.microsoft.com/azure/active-directory/fundamentals/active-directory-whatis) token authentication through a set of convenient [TokenCredential](https://docs.microsoft.com/javascript/api/@azure/core-auth/tokencredential) implementations. It enables Azure SDK clients to authenticate with AAD, while also allowing other JavaScript and TypeScript apps to authenticate with AAD work and school accounts, Microsoft personal accounts (MSA), and other Identity providers through the [AAD B2C](https://docs.microsoft.com/azure/active-directory-b2c/overview) service.
```

Reply:
woo! perfect! Thank you!